dali/test/python/test_external_source_impl.py

```diff
@@ -416,26 +416,6 @@ def iter_setup(self):
             break
     assert(iter_num == i)
 
-def test_external_source_fail():
-    class ExternalSourcePipeline(Pipeline):
-        def __init__(self, batch_size, external_s_size, num_threads, device_id):
-            super(ExternalSourcePipeline, self).__init__(batch_size, num_threads, device_id)
-            self.input = ops.ExternalSource()
-            self.batch_size_ = batch_size
-            self.external_s_size_ = external_s_size
-
-        def define_graph(self):
-            self.batch = self.input()
-            return [self.batch]
-
-        def iter_setup(self):
-            batch = datapy.zeros([self.external_s_size_,4,5])
-            self.feed_input(self.batch, batch)
-
-    batch_size = 3
-    pipe = ExternalSourcePipeline(batch_size, batch_size - 1, 3, 0)
-    pipe.build()
-    assert_raises(RuntimeError, pipe.run)
```

Comment:
Why was it removed? I think this still holds.

Reply:
This test check, if an exception is thrown when external source is fed with different batch sizes. We do not really want this scenario to throw an exception

Reply:
OK, whatever, I'm not particularly attached to this test, however, the exception should still be thrown until we're reasonably sure that feeding batches of varying length actually works.

Reply:
I would keep this  test in this PR, and remove when it actually doesn't apply anymore.